### PR TITLE
Consistency fixups across all logging endpoints

### PR DESCRIPTION
**Pull request by kellymclaughlin**
* Address some spelling and grammer issues in comments

### CODE REVIEW

**Comment by Integralist** [comment]
👋🏻 @kellymclaughlin as discussed in your other PR it looks like ServiceID shouldn't be a required flag because the internal implementation is first checking the flag, then for the value within a configuration file before erroring if no value exists in either of those places:

https://github.com/fastly/cli/blob/c0f337321376bb029c1ffee8b8c1181289dff695/pkg/compute/manifest/manifest.go#L52-L63

But your other tweaks might still be fine to be reviewed. I'll let you revert the ServiceID changes first before I review what's left.

Thanks!

**Comment by kellymclaughlin** [comment]
@Integralist I made service-id optional again and so this should be ready for review now.

File: pkg/logging/splunk/update.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ type UpdateCommand struct {
 	manifest manifest.Data
 
 	// required
-	EndpointName string // Can't shaddow common.Base method Name().
```
Comment (Integralist) [inline pkg/logging/splunk/update.go:20]: @kellymclaughlin looks like you missed out one instance:

```
// UpdateCommand calls the Fastly API to update Splunk logging endpoints.
```

Did you leave this in just as a way to ensure I reviewed every single file? 😉 well... I did check _every single file_ lol 😆

**Review by Integralist — COMMENTED** [review]
👋🏻 @kellymclaughlin just one small addition please and then this is good to go 🙂

File: pkg/logging/splunk/update.go (hunk 1 above)
Comment (kellymclaughlin) [reply pkg/logging/splunk/update.go:20]: Darn it. How did I miss that? :facepalm:

Comment (kellymclaughlin) [reply pkg/logging/splunk/update.go:20]: Fixed

**Review by Integralist — APPROVED** [review]
LGTM. Thanks Kelly 🎉

**Pull request merged**